### ISSUE TITLE
Infinite scroll limited to 3-5 pages

### ISSUE THREAD

**Issue by narcolepticinsomniac** [HorzaGobuchul/Amateur-Voat-Enhancements#11]
I'm not sure if this is an AVE issue or if Voat isn't cooperating, but infinite scroll rarely returns more than 3-5 pages before returning "Sorry, I couldn't find more content".

**Comment by HorzaGobuchul** [comment]
I think this is a Voat issue. 
It only happens with your front page when logged in right?

**Comment by narcolepticinsomniac** [comment]
Yeah, the front page. I don't know if it happens elsewhere, I'm not all that adventurous. Log-in status doesn't seem to be a factor though, I hardly ever log in.

I haven't spent much time there lately (it's gotten way too political for me), but I do still scroll through a handful of pages of top posts. Infinite scroll used to work. The site has been performing shittier as a whole, so maybe that's the cause.

**Comment by HorzaGobuchul** [comment]
I replicated the bug without AVE. Infinite scroll works fine, after page 4 there is simply no more content to load.

I think the front page sorting algorithm is finite in some way. I'll dig into Voat's code and see if something can be done.
